File: python/examples/get_registered_agent_products.py
import pprint
import os
import sys

sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

from request import BaseRequest
from dotenv import dotenv_values

config = dotenv_values()

# Example of GET /registered-agent-products

class GetRegisteredAgentProductsRequest(BaseRequest):
    def __init__(self):
        super().__init__()
    
    def get_products(self, website_url):
        params = {
            'url': website_url
        }
        return self.make_request('GET', '/registered-agent-products', params=params)

website_url = config['WEBSITE_URL']

request = GetRegisteredAgentProductsRequest()
response = request.get_products(website_url)

pprint.pprint(response)

File: javascript/examples/get_registered_agent_products.js
const baseRequest = require('../base_request.js').baseRequest
const path = require('path');
process.chdir(path.resolve(__dirname, '../../'));
require('dotenv').config();

// Example of GET /registered-agent-products

const WEBSITE_URL = process.env.WEBSITE_URL;

params = {
	url: WEBSITE_URL
}

token = baseRequest.request.token({ path: '/registered-agent-products' });

baseRequest.request.get({ path: '/registered-agent-products', token: token, queryParams: params });